### PR TITLE
[FEAT] Add strict validation config key

### PR DESCRIPTION
**Pull request by Augustinio**
### Description

- Added `strict_validation` key to emulators config.
- Updated version, readme and changelog

### Checklist

- [x] The title of the PR follows the right format: [{Label}] {Short Message}. Label examples: IMPROVEMENT, FIX, REFACTORING... Short message is about what your PR changes.

#### Versioning (PASQAL developers only)

- [x] Update the version of pasqal-cloud in `_version.py` following the changes in your PR and by using [semantic versioning](https://semver.org/).

#### Documentation

- [x] Update CHANGELOG.md with a description explaining briefly the changes to the users.

#### Tests

- [x] Unit tests have been added or adjusted.
- [x] Tests were run locally.

#### Internal tests pipeline (PASQAL developers only)
- [ ] Update and run the internal tests while targeting the branch of this PR.
 If your PR hasn't changed any functionality, it still needs to be validated against internal tests.

#### After updating the version (PASQAL developers only)

- [ ] Open a PR on the internal tests that updates the version used for the pasqal-cloud backward compatibility tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,22 @@ print(f"workload-id: {workload.id}, status: {workload.status}, result: {workload
 Some emulators, such as EMU_TN and EMU_FREE, accept further configuration to control the emulation.
 This is because these emulators are more advanced numerical simulation of the quantum system.
 
+For both emulators, you may choose to validate the sequence and jobs as though they were running
+on the FRESNEL qpu with the `strict_validation` key of the configuration. Defaults to False.
```
Comment (MatthieuMoreau0) [inline README.md:228]:
```suggestion
By default, validation rules are more permissive for jobs targeting an emulator than on the Fresnel QPU when submitting jobs to the cloud platform. 
You may however wish to validate that your job running on an emulator is compatible with Fresnel.
To that extent, set the `strict_validation` key in the configuration to `True`. Defaults to False.
```

File: pasqal_cloud/device/configuration/base_config.py [hunk 2]
```diff
@@ -34,6 +34,8 @@ class BaseConfig:
     parameters are added to the configuration dictionary.
     """
 
+    strict_validation: bool = False
+    """Whether to validate an emulator job as though it were for FRESNEL."""
```
Comment (MatthieuMoreau0) [inline pasqal_cloud/device/configuration/base_config.py:38]: Is `as though` correct in that context? It reads funny to me but maybe I'm not aware of this formulation in English

**Review by MatthieuMoreau0 — APPROVED** [review]
Couple docs suggestions. Thanks for the changes!

File: pasqal_cloud/device/configuration/base_config.py (hunk 2 above)
Comment (Augustinio) [reply pasqal_cloud/device/configuration/base_config.py:38]: as though means as if
https://www.collinsdictionary.com/dictionary/english/as-though#:~:text=phrase,he'd%20said%20something%20funny.

Comment (MatthieuMoreau0) [reply pasqal_cloud/device/configuration/base_config.py:38]: TIL :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.8.0] - 2024-04-15
+
+### Added
+
+- Added `strict_validation` option to emulators configuration.
```
Comment (Mildophin) [inline CHANGELOG.md:9]: english nitpick: emulator*

**Comment by Mildophin** [comment]
LGTM, rebase needed to allow me to approve it apparently. But here we go : ✅

**Pull request merged**